extension/view/js/InPageTranslation.js

```diff
@@ -112,21 +311,24 @@ class InPageTranslation {
          */
         this.translationsCounter += 1;
 
+        // Debugging: mark the node so we can add CSS to see them
+        node.setAttribute("x-bergamot-translated", this.translationsCounter);
```

Comment:
Do we need to add CSS for the case when QE is not selected by user? Based on my understanding, this code is written assuming QE is always on. We need to refactor the whole file to mark the nodes to add CSS only when QE is on.

Reply:
This is debugging CSS. Please use the extension.

![image](https://user-images.githubusercontent.com/727292/155732207-add46d7e-8a2a-4d05-b270-a1bbe876d734.png)

Reply:
Which extension are you talking about?

Reply:
https://github.com/jelmervdl/firefox-translations/releases/tag/v0.2.6 ([source code](https://github.com/jelmervdl/firefox-translations))